src/CMakeLists.txt

```diff
@@ -139,49 +132,30 @@ set(SOURCES
     Timer.cpp
     ${CMAKE_CURRENT_BINARY_DIR}/git_commit_current.cpp)
 
-set(tgt __objects)
-add_library(${tgt} OBJECT ${SOURCES} ${INCLUDES})
-target_include_directories(${tgt} PRIVATE ${path_include_common})
-target_include_directories(${tgt} SYSTEM PRIVATE ${path_include_common_suppressed_warnings})
-
+add_library(forwarder-library ${SOURCES} ${INCLUDES})
```

Comment:
I can see why the automatic registration of modules stopped working. You no longer use an `OBJECT` library. That is fine with me though.